lib/pages/caregiver/awards_page.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:fokus/data/model/user/caregiver.dart';
+import 'package:fokus/utils/theme_config.dart';
```

Comment:
Many unused imports clogging the Dart Analysis - it's good practice to keep it clear as it can prove useful in showing some flaws in code.